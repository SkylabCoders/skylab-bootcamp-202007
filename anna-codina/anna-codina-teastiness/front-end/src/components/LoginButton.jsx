import React from 'react';
import { useAuth0 } from '@auth0/auth0-react';

const LoginButton = () => {
  const { loginWithRedirect, isAuthenticated } = useAuth0();

  return (
    !isAuthenticated && (
      <button className="auth-button" onClick={() => loginWithRedirect()}>
        Log In
      </button>
    )
  );
};

export default LoginButton;
