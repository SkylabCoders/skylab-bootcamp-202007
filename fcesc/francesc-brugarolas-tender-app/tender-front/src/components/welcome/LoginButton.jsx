import React from 'react';
import './signin.sass';
import { useAuth0 } from "@auth0/auth0-react";

const LoginButton = () => {
  const { loginWithRedirect } = useAuth0();

  return <button className='btn btn-default btn-centered' onClick={() => loginWithRedirect()}>Log In</button>;
};

export default LoginButton;