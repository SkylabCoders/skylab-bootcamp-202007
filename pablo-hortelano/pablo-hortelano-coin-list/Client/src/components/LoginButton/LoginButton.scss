.login-anchor {
  font-size: 1rem;
  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
  font-weight: 400;
  line-height: 1.5;
  letter-spacing: 0.00938em;
  color: #2f00ee;
}
