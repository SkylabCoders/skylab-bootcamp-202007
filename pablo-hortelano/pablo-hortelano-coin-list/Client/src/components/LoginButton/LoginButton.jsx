import React from "react";
import { useAuth0 } from "@auth0/auth0-react";
import "./LoginButton.scss";

const LoginButton = () => {
  const { loginWithRedirect } = useAuth0();

  return (
    <button
      className="login-anchor pointer"
      onClick={() => loginWithRedirect()}
    >
      Log In
    </button>
  );
};

export default LoginButton;
