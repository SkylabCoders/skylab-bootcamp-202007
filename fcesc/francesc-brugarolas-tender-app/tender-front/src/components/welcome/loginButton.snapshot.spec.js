import React from 'react';
import renderer from 'react-test-renderer';
import LoginButton from './LoginButton';

const tree = renderer.create(<LoginButton />).toJSON();

describe('LoginButton component snapshot', () => {
    test('should match', () => {
        expect(tree).toMatchSnapshot();
    })
})