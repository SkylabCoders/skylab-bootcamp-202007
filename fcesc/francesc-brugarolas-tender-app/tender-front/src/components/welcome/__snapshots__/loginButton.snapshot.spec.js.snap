// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton component snapshot should match 1`] = `
<button
  className="btn btn-default btn-centered"
  onClick={[Function]}
>
  Log In
</button>
`;
